site/plugin.md

```diff
@@ -114,4 +114,4 @@ If you are using `systemd` with Docker 1.9, it is advised that you modify the Do
 
 **See Also**
 
- * [How the Weave Network Plugin Works](/site/plugin/plugin-how-it-works.md)
+ * [How the Weave Network Plugin (Legacy) Works](/site/plugin/plugin-how-it-works.md)
```

Comment:
We should at least link to the v2 doc; probably also mention in this doc that there is a Swarm-compatible option.